mrmustard/physics/__init__.py

```diff
@@ -13,121 +13,28 @@
 
-    Args:
-        A (State): the quantum state
+The internal engine of MM is powered by the computation of the quantum circuits between Fock and Bargmann representations. The theory about Fock and Bargmann representations is illustrated in xx and xx.
+The most important computation between representations is the inner product, which can be realized by gaussian integrals.
 
-    Returns:
-        float: the norm of the state
-    """
-    if A.is_gaussian:
-        return A._norm
-    return fock.norm(A.fock, is_dm=A.is_mixed)
+Under the level of the representation, the data structure has been defined to store the information to describe the quantum object, which is called ``Ansatz``. 
+``Ansatz`` is not only the data, but also includes the basic mathematical operations of the data. Each ``Representation`` has an attribue ``Ansatz``.
 
+For a quantum object, it is also powerful to be able to convert between different representations. We support the conversion from Bargmann to Fock right now for all quantum objects. Especially, we also support the transformations from/to s-parametrized phase space representation and from/to quadrature representation for quantum states.
 
-def overlap(A, B) -> float:
-    r"""Calculates the overlap between two quantum states.
+Check out our guides to learn more about :mod:`~mrmustard.physics` and its core functionalities:
 
-    If the states are both pure it returns :math:`|<A|B>|^2`, if one is mixed it returns :math:`<A|B|A>`
-    and if both are mixed it returns :math:`Tr[AB]`.
-
-    Args:
-        A (State) the first quantum state
-        B (State) the second quantum state
-
-    Returns:
-        float: the overlap between the two states
-    """
-    raise NotImplementedError
-
-
-def von_neumann_entropy(A) -> float:
-    r"""Calculates the Von Neumann entropy of a quantum state.
-
-    Args:
-        A (State) the quantum state
-
-    Returns:
-        float: the Von Neumann entropy of the state
-    """
-    if A.is_gaussian:
-        return gaussian.von_neumann_entropy(A.cov)
-    return fock.von_neumann_entropy(A.fock, a_dm=A.is_mixed)
-
-
-def relative_entropy(A, B) -> float:
-    r"""Calculates the relative entropy between two quantum states.
-
-    Args:
-        A (State) the first quantum state
-        B (State) the second quantum state
-
-    Returns:
-        float: the relative entropy between the two states
-    """
-    raise NotImplementedError
-
-
-def trace_distance(A, B) -> float:
-    r"""Calculates the trace distance between two quantum states.
-
-    Args:
-        A (State) the first quantum state
-        B (State) the second quantum state
+* The :mod:`~mrmustard.physics.ansatze` guide introduces the concept of an ansatz and two pre-defined ansatze.
+* The :mod:`~mrmustard.physics.representations` guide how to initialize representations and two basic representations: :class:`~mrmustard.physics.representations.Fock` and :class:`~mrmustard.physics.representations.Bargmann`.
```

Comment:
two basis representations? -no that does not make sense
But I would not consider Fock and in particular Bargmann for basic.